liferay-tango/liferay-portal
# LPS-168787 HR_494 SR should read "User Segment" icon with purpose info and named as a button like "Add User Segment Button"

Motivation
=======
This PR contains the changes to make the Add Segment button in the Segments Entry Menu more informative from an Accessibility perspective.
[Link to the story](https://issues.liferay.com/browse/LPS-168787)

Proposed Solution
========
The solution consists in changing the key used from 'User Segment' to 'Add New User Segment' to describe the button.

Steps to Verify:
----------------
1. Use JAWS
2. Navigate to Product menu > People > Segments
3. Navigate to the User Segment "+" icon and assert the SR announces
4. Hover over the button and check the label is 'Add New User Segment'

Screenshots:
----------------
![Screenshot 2023-01-17 at 15 49 52](https://user-images.githubusercontent.com/91207948/212930283-8df8b6c3-2739-4982-a50c-392f6f903d40.png)
